gpt_index/__init__.py

```diff
@@ -73,6 +73,7 @@
     TwitterTweetReader,
     WeaviateReader,
     WikipediaReader,
+    ObsidianReader,
```

Comment:
minor note:
I think the imports are sorted in alphabetical order.

You can sort imports automatically in VsCode:
https://eshlox.net/2019/12/02/vscode-sort-python-imports-automatically